BenchmarksCore/BenchmarksCore.csproj

```diff
@@ -2,11 +2,9 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFramework>net7.0</TargetFramework>
     <ServerGarbageCollection>true</ServerGarbageCollection>
-    <ConcurrentGarbageCollection>true</ConcurrentGarbageCollection>
-    <RetainVMGarbageCollection>true</RetainVMGarbageCollection>
-    <TieredCompilation>false</TieredCompilation>
```

Comment:
The reason tiered compilation was off is that in the past it often caused different results between runs. Also some profilers had issues with that. 
This is just a default that can be changed locally when doing benchmarking. I think it is ok either way.

Reply:
Hopefully it should be fairly stable now. Another problem with disabling it is getting not-representative performance (`isValueAtomicPrimitive` not being optimized away with dead branches completely removed).

The benchmark suite could be rewritten to use BenchmarkDotNet if necessary, which would also help with ensuring stability.

The only issue I have no solution for is, even in BDN, on Windows some benchmarks are notoriously bimodal which is, as I assume, caused by GC + thread migration.